### ISSUE TITLE
Question from the clouds

### ISSUE THREAD

**Issue by kimocoder** [acai66/Acai-Oneplus6-or-Oneplus6T-Nethunter-kernel#1]
Hi.

First off, let's say it's sad the kernel sources isn't 'shared' or available, as we've could have collaborated on it. And it loses a couple percents trust when it's not, but that's ok :+1: 

Good job on the kernel. btw,
Do you wan't your kernel to be the official NetHunter kernel?

I have a couple notes ..
* bootkali, killkali etc.. the "scripts" is added in the AnyKernel3 package at the 'xbin', breaking newer releases of the NetHunter app itself, as it forces down. It should be avoided, I'll add those scripts to the 'nethunter-installer' backend instead, so the chroot will be available for ADB also (probably as your though was?)

* Which patches is added to the kernel?

**Comment by acai66** [comment]
source code available here
https://github.com/acai66/op6_kernel
I'm rookie on porting nethunter kernel, and i dont know how to fix errors or bugs, but i really want  to get official support.
only patch mac80211 frame injection and rtl8812au driver.

**Comment by kimocoder** [comment]
Yeah, I can add not HID and injection patches. There are some missing I see there. rtl8812au yeah, I'm quite familiar with that one 😆

**Comment by acai66** [comment]
haha

**Comment by kimocoder** [comment]
![IMG_20200201_194441](https://user-images.githubusercontent.com/4252297/73597525-2bdd7500-452d-11ea-8f5f-b27532482791.jpg)

![IMG_20200201_194518](https://user-images.githubusercontent.com/4252297/73597526-2bdd7500-452d-11ea-9f49-0b4c149e8e18.jpg)

**Comment by kimocoder** [comment]
You'll find a more updated app below, the NetHunter store also.

http://kimocoder.aircrack-ng.org/

![Screenshot_20200201-235158](https://user-images.githubusercontent.com/4252297/73600266-eb422380-454d-11ea-81ea-41f85a4d97be.jpg)

![Screenshot_20200201-235144](https://user-images.githubusercontent.com/4252297/73600267-eb422380-454d-11ea-9005-5a02b02f6d55.jpg)

**Comment by acai66** [comment]
> You'll find a more updated app below, the NetHunter store also.
> 
> http://kimocoder.aircrack-ng.org/
> 
> ![Screenshot_20200201-235158](https://user-images.githubusercontent.com/4252297/73600266-eb422380-454d-11ea-81ea-41f85a4d97be.jpg)
> 
> ![Screenshot_20200201-235144](https://user-images.githubusercontent.com/4252297/73600267-eb422380-454d-11ea-9005-5a02b02f6d55.jpg)

wonderful

**Comment by Droid-MAX** [comment]
Does the V10 version of the kernel do not have the driver of the FT2232H series chip?

**Comment by acai66** [comment]
> Does the V10 version of the kernel do not have the driver of the FT2232H series chip?

![op6_kernel_ftdi](https://user-images.githubusercontent.com/37766207/90274882-f1697300-de93-11ea-8818-57f2e1f24caf.png)
I've enabled ` USB FTDI Single Port Serial Driver` , so `FT2232H` should work fine.
